content/en/cd-as-a-service/tasks/cli.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Install and Upgrade the CD-as-a-Service CLI
+linkTitle: Install/Upgrade CLI
+description: >
+  Install the Armory Continuous Deployment-as-a-Service CLI natively on Linux or Mac or use a Docker image. 
+---
+
+## CLI overview
+
+You can install the CLI (`armory`) on your Mac or Linux workstation, or you can run the CLI in Docker. 
```

Comment:
Its possible to install the CLI on windows by downloading the CLI from the releases page. 
https://github.com/armory-io/armory-cli/releases/tag/v1.14.5